src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -344,7 +344,7 @@ public static class Sse
         /// <summary>
         /// __m128 _mm_rcp_ss (__m128 a)
         /// </summary>
-        public static Vector128<float> ReciprocalScalar(Vector128<float> value) => ReciprocalScalar(value);
+        public static Vector128<float> ReciprocalScalar(Vector128<float> upper, Vector128<float> value) => ReciprocalScalar(upper, value);
```

Comment:
This leads to confusion where the C++ intrinsic doesn't include the additional argument, and makes me wonder whether we should include *both* the C++intrinsic as well as the actual opcode, and use the parameter names, e.g. for this case we would have:
```
         /// <summary>
          /// __m128 _mm_rcp_ss (__m128 value)
          /// rcpps upper, value
          /// </summary>
```
Thoughts?

Reply:
There are only 3 intrinsics which differ (`rcp_ss`, `rsqrt_ss`, and `sqrt_ss`), we could probably just add an explicit comment.
```
/// <summary>
/// __m128 _mm_rcp_ss (__m128 value)
/// We deviate from the native signature and allow the user to specify upper, rather than having it implicitly come from value. This is for consistency with the other scalar intrinsics.
/// </summary>
```